buildpacks/python/go.mod

```diff
@@ -47,5 +48,8 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/apimachinery v0.20.7 // indirect
 )
+
+replace kn-fn/shared v0.0.0 => ../_shared
```

Comment:
Again, this doesn't feel "right" to me.

Reply:
To be specific, are you referring to the semver?

Reply:
Referring to the need for replace with a relative path.

Reply:
I understand, since typically `replace` statements are not ever checked in. In my research for how to share projects like this, this was the common pattern (and exception to the not-checked-in rule of thumb).

Reply:
I guess it comes down to whether we like this more than having to maintain a separate repo (which I gather is something we really want to avoid)

Reply:
There's very minimal code right now that it's probably not worth throwing into its own repo.

Reply:
Did not mean to make a separate repo, was just wondering if there's a different way to handle `replace`.

Reply:
Yeah figured you didn't want that either 🙂 . I meant that from what I've researched, that's the only other option so I think we're stuck with `replace` for this use case. I could be wrong, but I haven't seen any other ways thus far.